.github/workflows/unit.yml

```diff
@@ -0,0 +1,26 @@
+name: Go Unit CI
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        go: [1.13, 1.14]
+        os: [ubuntu-latest, macos-latest] # windows-latest doesn't support find -wholename
```

Comment:
We should find a windows-friendly way to do gofmt

Reply:
Yup.
The goal of the PR isn't to add OS' or runtime versions, just split the CI.

Created issue:
https://github.com/GoogleCloudPlatform/functions-framework-go/issues/50